invenio/legacy/oaiharvest/admin.py

```diff
@@ -243,14 +243,14 @@ def perform_request_editsource(oai_src_id=None, oai_src_name='',
                 oai_src_post = []
 
             oai_src = OaiHARVEST.query.get(oai_src_id)
-            oai_src.setspecs = " ".join(oai_src_sets)
-            oai_src.baseurl = oai_src_baseurl
-            oai_src.name = oai_src_name
-            oai_src.metadataprefix = oai_src_prefix
-            oai_src.postprocess = "-".join(oai_src_post)
+            oai_src.setspecs = " ".join(oai_src_sets).strip()
```

Comment:
maybe it would be better to create validators/setters for the fields in model, wdyt?